api/lib/infrastructure/repositories/certification-report-repository.js

```diff
@@ -26,34 +27,35 @@ module.exports = {
     return _.map(certificationCourses, CertificationReport.fromCertificationCourse);
   },
 
-  async finalize({ certificationReport, transaction = undefined }) {
-    const saveOptions = { patch: true, method: 'update' };
-    if (transaction) {
-      saveOptions.transacting = transaction;
-    }
-
-    await new CertificationCourseBookshelf({ id: certificationReport.certificationCourseId })
-      .save({ hasSeenEndTestScreen: certificationReport.hasSeenEndTestScreen }, saveOptions);
-
-    if (certificationReport.examinerComment) {
-      await new CertificationIssueReportBookshelf({
-        certificationCourseId: certificationReport.certificationCourseId,
-        description: certificationReport.examinerComment,
-        category: CertificationIssueReportCategories.OTHER,
-      }).save(null, { transacting: transaction });
-    }
-  },
-
   async finalizeAll(certificationReports) {
     try {
       await Bookshelf.transaction((trx) => {
-        return Promise.all(certificationReports.map((certificationReport) => {
-          return this.finalize({ certificationReport, transaction: trx });
-        }));
+        return bluebird.mapSeries(certificationReports, (certificationReport) => {
+          return _finalize({ certificationReport, transaction: trx });
+        });
       });
     } catch (err) {
       throw new CertificationCourseUpdateError('An error occurred while finalizing the session');
     }
   },
 
 };
+
+async function _finalize({ certificationReport, transaction = undefined }) {
+  const saveOptions = { patch: true, method: 'update' };
+  if (transaction) {
+    saveOptions.transacting = transaction;
+  }
+
+  await new CertificationCourseBookshelf({ id: certificationReport.certificationCourseId })
+    .save({ hasSeenEndTestScreen: certificationReport.hasSeenEndTestScreen }, saveOptions);
+
+  // Remove this behavior when FT_REPORTS_CATEGORISATION is removed
```

Comment:
// Remove this behavior when FT_REPORTS_CATEGORISATION is removed (also search for examinerComment)